setup.cfg

```diff
@@ -71,25 +72,25 @@ doc =
     sphinx-bootstrap-theme
     sphinx-rtd-theme
 full =
-    astropy>=4.0.1,<5.0.0
+    astropy>=4.0.1,<6.0.0
     f90nml>=1.1.2
-    fastcache~=1.0.2
+    fastcache>=1.0.2
     firefly-vis>=2.0.4,<3.0.0
-    glueviz~=0.13.3
     h5py>=3.1.0,<4.0.0
```

Comment:
That's because the transition from 2 to 3 wasn't smooth for us (stuff broke that needed refactoring on our side), and because it's an optional deps, I assume we don't want to let it break our CI next time they do a breaking change, most likely in h5py 4.0
This is the same reason why firefly-vis has an upper limit. Mostly the specs are more strict where upstream breaks often.

Reply:
OK!